bokehjs/src/lib/core/ui_events.ts

```diff
@@ -262,6 +266,11 @@ export class UIEvents implements EventListenerObject {
 
         const active_inspectors = this.toolbar.inspectors.filter(t => t.active)
         let cursor = "default"
+        let toolbar_is_visible: boolean =  false
```

Comment:
Type annotation isn't necessary for local variables with primitive types. Also, bad whitespace.